src/app/heading/heading.component.html

```diff
@@ -1,26 +1,15 @@
 <header class="header">
     <div class="header__logo-box">
-        <img src="../../assets/img/turingLogo.png" alt="Logo" class="header__logo">
+        <img src={{logo}} alt="Logo" class="header__logo">
     </div>
     <div class="header__text-box">
-        <h1 class="heading-primary">Turing Index</h1>
+        <h1 class="heading-primary">{{title}}</h1>
+    </div>
+    <div class="header__button-box">
```

Comment:
I think a possible idea we could explore in line of what the PM was talking about keeping the map interactive might be to place these buttons on the map? That way it's almost an implicit instruction that this map is interactive